code/extensions/Lumberjack.php

```diff
@@ -147,4 +147,20 @@ protected function shouldFilter() {
 			&& in_array($controller->getAction(), array("treeview", "listview", "getsubtree"));
 	}
 
+	/**
+	 * Checks config for a specified child_classname on the class extending Lumberjack.
+	 * Uses SiteTree if none found in config.
+	 *
+	 * @return string
+	 */
+	protected function getChildClassName() {
+		$childClassName = "SiteTree";
+		if ($childClassNameConfig = Injector::inst()->create($this->owner->ClassName)->config()->child_classname) {
+			if (class_exists($childClassNameConfig)) {

```

Comment:
`Config::inst()->get($this->owner->ClassName, 'child_classname` is a bit more concise